### PR TITLE
Engage: Added missing image to Headless article

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
Engage v13

## Deadline (if relevant)

Anytime

### CODE REVIEW

File: 13/umbraco-engage/developers/headless/README.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ description: >-
...
 
-Umbraco Engage has a **Umbraco.Engage.Headless** package that can be installed to integrate with Umbraco 12.0+. Headless Content Delivery API, enabling personalized content, A/B tests, and segmentation.
+Umbraco Engage offers the **Umbraco.Engage.Headless** package for seamless integration with Umbraco 12.0 and later. This package enables access to the Headless Content Delivery API, enabling personalized content, A/B tests, and segmentation.
 
 ## Requirements
 
-To install Umbraco.Engage.Headless, ensure the following:
+To install Umbraco.Engage.Headless, ensure the following prerequisites:
 
 * Umbraco v12 or higher is required to integrate with the [Content Delivery API](https://docs.umbraco.com/umbraco-cms/reference/content-delivery-api).
```
Comment (sofietoft) [inline 13/umbraco-engage/developers/headless/README.md:15]: Umbraco Engage is only for 13 currently, so let's either get rid of this line, or update it to 13 💪 😁

File: 13/umbraco-engage/developers/headless/README.md [hunk 2]
```diff
@@ -62,4 +62,6 @@ builder.CreateUmbracoBuilder()
 
 3. Rebuild and run your site.
 4. Navigate to **/umbraco/swagger** in your browser.
-5. From the top right definition dropdown, check for the **Umbraco Engage Marketing API.**
+5. From the top right definition dropdown, check for the **Umbraco Engage API**
```
Comment (sofietoft) [inline 13/umbraco-engage/developers/headless/README.md:65]:
```suggestion
5. Check for the **Umbraco Engage API** from the top right definition dropdown.
```

**Review by sofietoft — COMMENTED** [review]
I've added a couple of suggestion 💪

File: 13/umbraco-engage/developers/headless/README.md (hunk 1 above)
Comment (eshanrnh) [reply 13/umbraco-engage/developers/headless/README.md:15]: That's a good point. I'll update it to v13.

**Comment by eshanrnh** [comment]
Thanks for your review, @sofietoft 🙌 The PR should be good to go 💪

**Review by sofietoft — APPROVED** [review]

**Pull request merged**